app/controllers/admin/events_controller.rb

```diff
@@ -92,7 +95,27 @@ def update
       end
     end
 
-    def create; end
+    def create
+      @url = admin_conference_program_events_path(@conference.short_title, @event)
+      @users = User.all.order(:name)
+      @languages = @program.languages_list
+      @event.submitter = current_user
+      @event.state = :confirmed
```

Comment:
I think we should allow speakers to confirm their event manually, as confirmation of an event means a final acknowledgement that the event is accepted and the speaker will be coming. If needed, an organizer can always manually confirm events from events#index.